jaxlib/setup.py

```diff
@@ -30,6 +30,10 @@
 if cuda_version and cudnn_version:
   __version__ += f"+cuda{cuda_version.replace('.', '')}-cudnn{cudnn_version.replace('.', '')}"
 
+rocm_version = os.environ.get("JAX_ROCM_VERSION")
+if rocm_version:
+    __version__ += f".{rocm_version.replace('.', '')}"
```

Comment:
Can you do `+rocm` just like the cuda version above?

Reply:
@yashk2810 Hi Yahs, I pushed an update. Please review